config/cmip6.yaml

```diff
@@ -11,3 +11,7 @@ sources:
   - metadata_yaml: /g/data/xp65/admin/access-nri-intake-catalog/config/metadata_sources/cmip6-oi10/metadata.yaml
     path:
       - /g/data/oi10/catalog/v2/esm/catalog.json
+
+  - metadata_yaml: /g/data/xp65/admin/intake/metadata/cmip6_ig45/metadata.yaml
+    path:
+    - /g/data/ig45/catalog/v2/esm/catalog.json 
```

Comment:
If this is the path where the 'live' `metadata.yaml` is being stored, then I'd recommend the reference copy's name/path within the repo match, i.e. be `cmip6_ig45`.